healthcheck/healthcheck.py

```diff
@@ -181,3 +182,27 @@ def _assess_overall_status(self):
             return False
 
         return True
+
+
+def _file_exists(path):
+    """Return True if a file exists at `path` (even if it can't be read),
+    otherwise False.
+
+    This is different from os.path.isfile and os.path.exists which return
+    False if a file exists but the user doesn't have permission to read it.
+    """
+    try:
+        os.stat(path)
+        return True
+    except OSError as e:
+
```

Comment:
> I think the comment says a lot.

I like comments. I just think that comment before last "raise" is not very useful, because it's clear that if two conditions above were false, the original error is raised.